src/onboarding/ChooseYourAdventure.tsx

```diff
@@ -201,4 +201,9 @@ const styles = StyleSheet.create({
   skip: {
     color: colors.gray3,
   },
+  cardText: {
+    ...fontStyles.small500,
```

Comment:
fontStyles is deprecated, is there a typeScale that we can change this to?

Reply:
Fixed in ef9a13fcdf13bafe65b0157ffe726c283e14f681 and updated the description pictures.